eras/shelley/test-suite/src/Test/TestScenario.hs

```diff
@@ -3,7 +3,11 @@
 
 module Test.TestScenario where
 
-import Cardano.Prelude hiding (Option)
+import Cardano.Prelude
+  ( putTextLn,
+    show,
+  )
```

Comment:
There is no real reason to go through `Text` here. Just switch it to regular `putStrLn` and `show` from regular `Prelude`